build/Makefile.kits

```diff
@@ -355,4 +359,32 @@ else
 GetLinuxOS:
 endif
 
+# Build SELinux policy modules for scxagent-logrotate
```

Comment:
Will it be possible to move sepolicy definition to Makefile.pf.Linux as this is a Linux only feature?

Reply:
I have tried the same but it is not working. Also in past we have not taken this approach to add rules in these platform specific files. Hence I am going ahead with adding the sepolicy rules in Makefile.kits file.